src/java.base/share/classes/java/math/BigDecimal.java

```diff
@@ -79,17 +78,23 @@
  * {@code BigDecimal} arithmetic are broadly compatible with selected
  * modes of operation of the arithmetic defined in ANSI X3.274-1996
  * and ANSI X3.274-1996/AM 1-2000 (section 7.4).  Unlike those
- * standards, {@code BigDecimal} includes many rounding modes, which
- * were mandatory for division in {@code BigDecimal} releases prior
- * to 5.  Any conflicts between these ANSI standards and the
+ * standards, {@code BigDecimal} includes many rounding modes
```

Comment:
Missing period at the end of the sentence (after "rounding modes")?

Reply:
Quite right; will fix in the next push. Thanks.